test/csuite/timestamp_abort/main.c

```diff
@@ -187,23 +188,36 @@ thread_ts_run(void *arg)
             continue;
         prev_all_dur_ts = all_dur_ts;
 
+        /* Ensure the durable ts doesn't move equal to or beyond the oldest non-committed
+         * transaction. */
+        oldest_commit = WT_TS_MAX;
+        for (i = 0; i < td->info; i++) {
+            WT_ORDERED_READ(ts_temp, ts_data[i]);
```

Comment:
Can we just do ts_temp = ts_data[i]?

Reply:
What was the goal of adding a read-barrier after each assignment? How does that help?